### PR TITLE
Added support for paths, prefix and rewritePrefix

### PR DESCRIPTION
**Pull request by daofresh**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
Would you mind to add a new test instead? Also add docs! Thanks!

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
good work!

File: README.md [hunk 1]
```diff
@@ -152,8 +152,9 @@ This module has _partial_ support for forwarding websockets by passing a
 A few things are missing:
 
 1. forwarding headers as well as `rewriteHeaders`
-2. support for paths, `prefix` and `rewritePrefix`
-3. request id logging
+2. request id logging
+
+The `prefix` and `rewritePrefix` options are also supported
```
Comment (jsumners) [inline README.md:157]: This seems insufficient documentation. Please explain what they are and how to use them with examples.

Comment (mcollina) [reply README.md:157]: @daofresh ping

**Comment by mcollina** [comment]
@daofresh any update?

**Pull request closed without merging**